### DISCUSSION TITLE
Win64 WinMain C++ Samples

### DISCUSSION THREAD

**Discussion by r2d2Proton** [microsoft/WindowsAppSDK#2614] [General]
Are there any samples that start from WinMain with C/C++ and dynamically load the elements needed?

I really do not want any generated code or other voodoo acting behind the scenes. I would rather perform the steps pragmatically accounting for what is needed to get the WindowsAppSDK working, and WinUI without the need for XAML or a XAML engine.

Help is appreciated.

**Comment by castorix** [comment]
I had done a test as I posted in [this thread](https://github.com/microsoft/WindowsAppSDK/discussions/2233) (I mainly program in C/C++ since 80's, so I don't like generated code and I want to understand every line of code I type...)
But I had to copy many generated blocks of code to make it work and the resulting code was not great at all...

**Reply by r2d2Proton** [reply]
**Agree with everything you are saying.**
